batch/install-without-bom/pom.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
We will only be moving the snippets folder

Reply:
Got it. Thanks.
PTAL at the changes!